### ISSUE TITLE
[Bug]: Table row highlights are not visible in high contrast inverted colours

### ISSUE THREAD

**Issue by cjrace** [dfe-analytical-services/shinyGovstyle#197]
### Overview

Hover over row highlighting in reactable tables does not show in Windows' High Contrast mode.

### Steps to Reproduce

Go to the showcase app and view in Windows' High Contrast mode.

<img width="433" height="233" alt="Image" src="https://github.com/user-attachments/assets/76622e42-1aa2-4b57-b2ef-8c0df2294524" />

### Expected Behavior

Low vision comments:

> “The highlights on the table here do not appear in high contrast. This can make the table much harder to access for low vision users, the highlights provide a helpful tool for users to know where they are and to follow the rows or column of a table which can be hard for some low vision users to do.This could be improved by giving the table ighlights that also show up in high contrast.”

### Additional Information

Page 51 of the September 2025 accessibility audit report.
